src/common/ConfUtils.cc

```diff
@@ -122,7 +122,8 @@ int ConfFile::parse_file(const std::string &fname,
       return -EINVAL;
     }
   } catch (const fs::filesystem_error& e) {
-    if (fs::is_other(fname)) {
+    std::error_code ec;
+    if (fs::is_other(fname, ec) && !ec) {
```

Comment:
i think we need to check ec first, as, if ec is non-zero, the return value won't be a valid one anymore. so something like
```c++
if (auto is_other = fs::is_other(fname, ec); !ec && is_other) {
```

Reply:
If there is an error, `is_other` returns false (the `file_status` is set to 'unknown'), so it falls through. The `ec` was mostly just to stop the exception from being raised.

Reply:
> If there is an error, `is_other` returns false

ahh, right!